types/frisby/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Christopher E. Woodland <https://github.com/cwoodland>
 //                 Johnny Li <https://github.com/johnny4753>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.8
 
 /// <reference types='jest'/>
```

Comment:
I don't actually think this is needed - I can't find any evidence of `@types/jest` being used in this file, but will look into it further once I've got CI green.

Reply:
It seems that the original reference was to `jasmine`, and was because it was argumenting the global `jasmine` namespace.

However, while v2 of `@types/frisby` removed this argumenting, the reference was kept in because the author wanted to have `@types/jasmine` pulled in because the library used `jasmine`?

I can understand why, and am not against it, but I think it would be better to remove this reference since `frisby.js` doesn't list `jest` as a dependency (not even as a `peerDependency`) nor does it require/import it anywhere, and because people using TypeScript will need to install `@types/jest` anyway as they'll be consuming them directly.

Additionally, the source code of `frisby` still references `jasmine` rather than `jest`, and was originally switched to `jest` as the `jasmine` types were causing problems (#32191).

@cwoodland @johnny4753 thoughts?

Reply:
I've made #48935 to remove the reference.